gcp-function-http/src/main/java/io/micronaut/gcp/function/http/HttpFunction.java

```diff
@@ -80,9 +83,9 @@ private ServletHttpHandler<HttpRequest, HttpResponse> initializeHandler() {
             @Override
             protected ServletExchange<HttpRequest, HttpResponse> createExchange(HttpRequest request, HttpResponse response) {
                 final GoogleFunctionHttpResponse<Object> res =
-                        new GoogleFunctionHttpResponse<>(response, getMediaTypeCodecRegistry(), conversionService);
+                        new GoogleFunctionHttpResponse<>(response, conversionService);
                 final GoogleFunctionHttpRequest<Object> req =
-                        new GoogleFunctionHttpRequest<>(request, res, conversionService, applicationContext.getBean(BodyBuilder.class));
+                        new GoogleFunctionHttpRequest<>(request, res, conversionService, applicationContext.getBean(BodyBuilder.class), applicationContext.getBean(Executor.class, Qualifiers.byName(TaskExecutors.BLOCKING)));
```

Comment:
this bean lookup will run on every request, can you move it to the constructor like ConversionService